lib/src/base-styles/_nx-colors.scss

```diff
@@ -55,9 +55,11 @@
   --nx-color-alert-warning: var(--nx-swatch-orange-50);
   --nx-color-alert-error: var(--nx-swatch-red-40);
 
-  //NxBinaryDonutChart variables
-  --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
-  --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-teal-60);
+  --nx-color-progress-background: var(--nx-swatch-indigo-95);
+  --nx-color-progress-fill: var(--nx-swatch-teal-60);
+
+  //NxBinaryDonutChart
+  --nx-color-binary-donut-chart-outline: var(--nx-color-component-background);
```

Comment:
I believe the intent is that this will always be the component background color, as it is intended to be invisible when on tiles and similar components.  As such I don't think it needs a distinct variable, the `NxBinaryDonutChart` styles can use `--nx-color-component-background` directly

Reply:
> > I believe the intent is that this will always be the component background color, as it is intended to be invisible when on tiles and similar components. As such I don't think it needs a distinct variable, the NxBinaryDonutChart styles can use --nx-color-component-background directly.
> 
> This variable also isn't being used currently.

I removed it here: [e35ef17](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/e35ef1734248fa1ff14a8e8ec23954a2f0a0c49e)